### ISSUE TITLE
`disk-pasting-test` failure

### ISSUE THREAD

**Issue by gheber** [gheber/kenzo#139]
```
Failure Details:
 --------------------------------
 HMEQ-DISK-PASTING []: 
      Unexpected Error: #<UNDEFINED-FUNCTION HMEQ-DISK-PASTING {100526F083}>
The function CAT-9:HMEQ-DISK-PASTING is undefined...
 --------------------------------
```

**Comment by sphyynx** [comment]
HMEQ-DISK-PASTING  not  in  Kenzo-9.   Was in  Kenzo-8.   There  is  a
DISK-PASTING-HMEQ in  Kenzo-9 but which  in fact returns  a reduction,
not  a  homology-equivalence. When  it  is  possible, Kenzo-9  uses  a
reduction instead of an equivalence, better efficiency.

**Comment by gheber** [comment]
Fixed in dddbc7b.
